src/neo4j/neo4j-functions.js

```diff
@@ -94,59 +99,58 @@ export async function neighbourhood(id) {
 
 /**
  * @param { String } id in the form of "dbName:dbId", ex: "ncbigene:207"
+ * @param { Boolean } withComplexes default true to show complexes
  * @returns an array of nodes that are neighbours to the specified gene
  */
-export async function getNeighbouringNodes(id) {
-    let record = await neighbourhood(id);
+export async function getNeighbouringNodes(id, withComplexes = true) {
+    let record = await neighbourhood(id, withComplexes);
     if (record) {
         return _.uniqBy(record.map(row => {
             return row.get('m').properties;
-        }), node => node.id);
+        }), edge => edge.id);
     }
     return null;
 }
 
 /**
  * @param {*} id in the form of "dbName:dbId", ex: "ncbigene:207"
+ * @param { Boolean } withComplexes default true to show complexes
  * @returns an array of relationships leading away from/leading to the specified gene
  */
-export async function getInteractions(id) {
-    let record = await neighbourhood(id);
+export async function getInteractions(id, withComplexes = true) {
+    let record = await neighbourhood(id, withComplexes);
     if (record) {
-        return _.uniqBy(record.map(row => {
+        return record.map(row => {
             return row.get('r').properties;
-        }), edge => edge.id);
+        });
     }
     return null;
 }
 
-export async function neighbourhoodWithoutComplexes(id) {
-    const driver = getDriver();
-    let session;
-    let record;
-    try {
-        session = driver.session({ database: dbName });
-        let result = await session.executeRead(tx => {
-            return tx.run(giveConnectedInfoByGeneIdNoComplexes, { id: id });
-        });
-        if (result.records.length > 0) {
-            record = result.records;
-        } else {
-            record = null;
-        }
-    } finally {
-        await session.close();
+/**
+ * @param {*} id in the form of "dbName:dbId", ex: "ncbigene:207"
+ * @param { Boolean } withComplexes default true to show complexes
+ * @returns null if id not found in database, object with 2 fields (one for neighbouring nodes and one for
+ * edges) otherwise
+ */
+export async function neighbourhoodReadable(id, withComplexes = true) {
+    let record = await neighbourhood(id, withComplexes);
+    if (record) {
+        return {
+            neighbouringNodes: _.uniqBy(record.map(row => { return row.get('m').properties; }), edge => edge.id),
```

Comment:
Why not just `nodes`?

Reply:
I was probably thinking about my helper function, `getNeighbouringNodes`, but `nodes` make more sense. I'll go ahead and change it